browses.md

```diff
@@ -54,6 +54,22 @@ Error codes:
 
 **404** if the browse index doesn't exist
 
+## Browse indexes configured for metadata linking
+### Get a single index configured as browse links for the given metadata fields
+**/api/discover/browses/search/byFields?fields=<:field>**
+
+Example: <http://dspace7.4science.it/dspace-spring-rest/#http://dspace7.4science.it/dspace-spring-rest/api/discover/browses/search/byFields?fields=dc.contributor.author&fields=dc.creator>
```

Comment:
Super tiny thing... let's not use an old URL here.  This URL structure is incorrect (this is from back when the REST API used to have a `dspace-spring-rest` path)

You could just change this to:
`Example: https://{dspace.server.url}/api/discover/browses/search/byFields?fields=dc.contributor.author&fields=dc.creator`

Reply:
@tdonohue I resolved this. Thank you for your feedback.